src/services/requestService.js

```diff
@@ -0,0 +1,28 @@
+import { Request } from '../models';
+import Helper from '../utils/helpers';
+
+/**
+ * Trip request services
+ */
+export default class RequestService {
+  /**
+   * @param {object} details trip details
+   * @returns{void}
+   */
+  static async oneway(details) {
+    const { travelDate } = details;
+    const existingRequest = await Request.count({ where: { travelDate } });
+
+    if (existingRequest) throw new Error('You\'ve already booked this trip');
```

Comment:
We have a helper for error now. Please use that

Reply:
okay will do